package.json

```diff
@@ -46,7 +46,7 @@
     "@commitlint/cli": "^17.6.7",
     "@commitlint/config-conventional": "^17.6.7",
     "@types/jest": "^29.5.3",
-    "@types/node": "^20.4.7",
+    "@types/node": "^20.5.9",
```

Comment:
is this the main node version being updated? Might need to check compatibility with Azure

Reply:
This is just for type checking, i.e not the actual node version